login.html

```diff
@@ -0,0 +1,161 @@
+        </div>
+        
+        <div class="fright menu-links">
+        <spna style="color:black">Campaigns</span>
+            <span style="color:black">&nbsp;&nbsp;|&nbsp;&nbsp;
+            <a href="/ngos">NGOs</a>
+        </div>
+  
+    </h1>
+    <div class="fclear"></div>
+</div>
+
+<div id="row">
+
+  <header>
+          <div class="form">
+        
+        <ul class="tab-group">
+          <li class="tab active"><a href="#signup">Sign Up</a></li>
+          <li class="tab"><a href="#login">Log In</a></li>
+        </ul>
+        
+        <div class="tab-content">
+          <div id="signup">   
+            <h1>Sign Up</h1>
+            
+            <form action="javascript:void(0)">
+            
+            <div class="top-row">
+              <div class="field-wrap">
+                <label>
+                  First Name*
+
+                </label>
+                <input type="text" required autocomplete="off" />
+              </div>
+          
+              <div class="field-wrap">
+                <label>
+                  Last Name*
+
+                </label>
+                <input type="text"required autocomplete="off"/>
+              </div>
+            </div>
+
+            <div class="field-wrap">
+              <label>
+                Email Address*
+
+              </label>
+              <input type="email"required autocomplete="off"/>
+            </div>
+            
+            <div class="field-wrap">
+              <label>
+                Set A Password*
+
+              </label>
+              <input type="password"required autocomplete="off"/>
+            </div>
+            
+            <button type="submit" class="button button-block"/>Register</button>
+            
+            </form>
+
+          </div>
+          
+          <div id="login">   
+            <h1>Login</h1>
+            
+            <form action="javascript:void(0)">
+            
+              <div class="field-wrap">
+              <label>
+                Email Address*
+
+              </label>
+              <input type="email"required autocomplete="off"/>
+            </div>
+            
+            <div class="field-wrap">
+              <label>
+                Password*
+
+              </label>
+              <input type="password"required autocomplete="off"/>
+            </div>
+            
+            <p class="forgot"><a href="#">Forgot Password?</a></p>
+            
+            <button class="button button-block"/>Log In</button>
+            
+            </form>
+
+          </div>
+          
+        </div>
+        
+    </div>
+  </header>
+</div>
+
+
+<div class="footer">
+    <div class="padding10">
+        &copy;2015 We Care.
+    </div>
+</div>
+<script src="/js/jquery.js"></script>
+<script src="/js/handlebars.js"></script>
+<script src="/js/jquery.easing.js"></script>
+<script src="/js/waterfall.js"></script>
+<script src="/js/index.js"></script>
+<script src="/js/creative.js"></script>
```

Comment:
creative.js file used for implementing effects in HTML page For example :  smooth scrolling, fixed navbar,etc.